test/boost/sstable_compaction_test.cc

```diff
@@ -2616,7 +2619,9 @@ SEASTAR_TEST_CASE(sstable_scrub_segregate_mode_test) {
             // We expect the scrub with mode=srub::mode::abort to stop on the first invalid fragment.
             sstables::compaction_type_options::scrub opts = {};
             opts.operation_mode = sstables::compaction_type_options::scrub::mode::abort;
-            compaction_manager.perform_sstable_scrub(table->as_table_state(), opts).get();
+            try {
+                compaction_manager.perform_sstable_scrub(table->as_table_state(), opts).get();
+            } catch (sstables::compaction_aborted_exception& e) {}
```

Comment:
If the intention is to swallow this exception in must be more visible, with a comment explaining and documenting it, and in a separate patch of course.
e.g.
```
    try {
        compaction_manager.perform_sstable_scrub(table->as_table_state(), opts).get();
    } catch (sstables::compaction_aborted_exception& e) {
        // Ignored, since x y z.
    }
```

Reply:
After the changes the exception even **should** be caught here. So I just check whether it was swallowed, analogically like in other test cases.

Reply:
I don't understand

Reply:
In this case `perform_sstable_scrub` is performed on invalid sstable in abort mode, so an exception is expected. Thus I check whether it was caught.